### DISCUSSION TITLE
Will they be adding slash command support and if so, when?

### DISCUSSION THREAD

**Discussion by EvokerKing** [Rapptz/discord.py#6184] [Q&A]
Just read the title.

**Comment by DuckMasterAl** [comment]


**Reply by EvokerKing** [reply]
So if I get the development version, how do I make it into a slash command instead of a normal one, because in the example they use `!hello`?

**Reply by DuckMasterAl** [reply]
That comment was about replies, not slash commands, sorry!
There is currently no information about if/when Slash Commands will be in discord.py.
However, there are some other libraries that have support for them.

**Comment by 15696** [comment]
Probably not. It has a lot of limitations and would require a complete rework of the commands extension. (as said by polyjitter in issue #6149)

You can use this meanwhile:
https://github.com/Zomatree/discord.py/tree/slashcommands

![image](https://user-images.githubusercontent.com/50557523/103980677-8ddf2b00-513d-11eb-9c56-7e2195f6fdb3.png)

**Comment by RGBCube** [comment]
It is implemented, to use it you need the development version: `pip install git+https://github.com/Rapptz/discord.py`
